.github/workflows/nuget_package_push_prerelease.yml

```
@@ -0,0 +1,35 @@
+name: Nuget Develop Prerelease
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  pushprerelease:
+    runs-on: windows-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
+    steps:
+      - name: Get current time
+        uses: srfrnk/current-time@master
+        id: current-time
+        with:
+          format: YYYYMMDDHHMMSS
```

```suggestion
          format: YYYYMMDDHHmmss
```
See https://github.com/srfrnk/current-time#format and https://momentjs.com/docs/#/displaying/
- `MM` is month, we want `mm` = minutes
- `SS` is fractional second, we want `ss` = seconds

Both are wrong in the BO4E repo also ;)

---

9a3561d7157a35c8671acc27c7ced64d1dc6a528